python/cugraph/sampling/random_walks.pxd

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
None of the cython code is even building at the moment. I wonder if it would be a better idea to remove all of it and add it to another PR when it can be properly tested before being merged.

Reply:
Joseph is looking into that. I had to check in something for him to fix and help him understand the overall structure, even if it;s incomplete at the moment. But, yes, for the purpose of this PR I might remove that Cython side.

Reply:
You can use a dev branch for sharing incomplete code.

Reply:
Done. I removed the faulty Cython for now, while Joseph is looking into it, so that the rest of CI is unblocked.